src/newusers.c

```diff
@@ -1244,21 +1229,22 @@ int main (int argc, char **argv)
 				fprintf(stderr,
 					_("%s: line %d: homedir must be an absolute path\n"),
 					Prog, line);
-				errors++;
-				continue;
-			};
+				fail_exit (EXIT_FAILURE);
+			}
 			if (mkdir (newpw.pw_dir, mode) != 0) {
 				fprintf (stderr,
 				         _("%s: line %d: mkdir %s failed: %s\n"),
 				         Prog, line, newpw.pw_dir,
 				         strerror (errno));
-			} else if (chown (newpw.pw_dir,
-			                  newpw.pw_uid,
-			                  newpw.pw_gid) != 0) {
+				fail_exit (EXIT_FAILURE);
+			}
+			if (chown(newpw.pw_dir, newpw.pw_uid, newpw.pw_gid) != 0)
+			{
 				fprintf (stderr,
 				         _("%s: line %d: chown %s failed: %s\n"),
 				         Prog, line, newpw.pw_dir,
 				         strerror (errno));
+				fail_exit (EXIT_FAILURE);
```

Comment:
Here I can't think of a good reason for chown to fail, so adding a error case here is fine by me.